.custom-words.txt

```diff
@@ -158,14 +217,22 @@ subgraph
 subgraphs
 subquery
 subtractive
+textsf
 th
-topologies
 timestep
 timesteps
+tombstoning
+topologies
 tradeoff
+transative
```

Comment:
These are misspellings of `transitive` and `transitively`

Reply:
Ah, that's what I get for copy/pasting a bunch of words from the checker without manually checking all of them 😅 Thanks